fiatconnect-api.md

```diff
@@ -2223,6 +2224,25 @@ platform can be used to consume Fiat Connect services by providing their `mobile
 }
 ```
 
+#### 8.3.2.4. `IfscAccount`
+
+`IfscAccount` is an account schema that represents transfers for `INR` currency.
+
+```
+{
+	accountName: `string`,
+	institutionName: `string`,
+	ifsc: `string`,
+	accountNumber: `string`,
+	country: `string`,
```

Comment:
Are IFSC-based accounts available anywhere _other_ than India? Will a user ever be able to set this to anything other than `"IND"`/will the `allowedValues` field allow the user to do so?

Reply:
IFCS is the branch identifier used in India. 
`RATN0000012` is a sample. This string has to be known before a transaction to the account number can be initiated. 
IFSC is only applicable for INR transactions.